site/content/docs/terraform-provider/checks-groups.md

```diff
@@ -133,6 +134,33 @@ resource "checkly_check" "send-weekly-digest-v-2" {
 
 Upon applying your terraform configuration changes, you will be returned a read-only key value for the heartbeat ping token. The token is stored in your `tfstate` file.
 
+
+### Multistep API checks
+
+As with Browser checks, when constructing a Multistep API check it is possible to provide the script directly in-line, 
+but the recommended approach is to store scripts in separate files.
+
+For example, a Multistep check can look as follows:
```

Comment:
What about adding the same info box as cli, with the runtime information, and also a link to the full multistep doc?

Reply:
I'll add the warning. There is a link to all the different check types at the top of the page.